### PR TITLE
[DYN-81]feat: make personal info screen

### PR DESCRIPTION
**Pull request by liubaterehova**

### CODE REVIEW

File: src/components/PhoneNumberInput/PhoneNumberInput.jsx [hunk 1]
```diff
@@ -14,12 +14,13 @@ function PhoneNumberInput({
...
   onChangePhone,
   onChangeFormattedPhone,
+  onBlur,
   error,
   text,
 }) {
   return (
     <View style={styles.container}>
-      {text && <Text style={styles.label}>{text}</Text>}
+      {text ? <Text style={styles.label}>{text}</Text> : null}
```
Comment (arlovaalina) [inline src/components/PhoneNumberInput/PhoneNumberInput.jsx:23]: Why do you need to add `null`?

File: src/components/PhoneNumberInput/PhoneNumberInput.styles.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export default StyleSheet.create({
   inputContainer: {
     borderRadius: 10,
     borderColor: colors.inputStandardBorder,
-    borderWidth: StyleSheet.hairlineWidth,
+    borderWidth: 1,
```
Comment (arlovaalina) [inline src/components/PhoneNumberInput/PhoneNumberInput.styles.js:36]: What was wrong with `StyleSheet.hairlineWidth`?

File: src/screens/PersonalInformation/PersonalInformation.jsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+        style={styles.wrapper}
+      >
+        {isLoading && <FullScreenLoader />}
+        <ScrollView
+          style={styles.wrapper}
+          contentContainerStyle={styles.container}
+        >
+          <View style={styles.wrapper}>
+            <MainHeader title="Personal Information" navigation={navigation} />
```
Comment (arlovaalina) [inline src/screens/PersonalInformation/PersonalInformation.jsx:116]: Do not pass `navigation` object directly, better use `useNavigation` hook instead.

File: package.json [hunk 4]
```diff
@@ -44,6 +47,7 @@
     "react-native-status-bar-height": "2.6.0",
     "react-native-svg": "12.1.0",
     "react-native-svg-transformer": "0.14.3",
+    "react-native-vector-icons": "8.0.0",
```
Comment (arlovaalina) [inline package.json:50]: Do you use this package somewhere?

**Review by arlovaalina — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (liubaterehova) [reply package.json:50]: yeah
this lib (https://www.npmjs.com/package/react-native-dropdown-picker )uses it. And we mist save this lib as I understand. I tried not to do it and have error

File: src/components/PhoneNumberInput/PhoneNumberInput.styles.js (hunk 2 above)
Comment (liubaterehova) [reply src/components/PhoneNumberInput/PhoneNumberInput.styles.js:36]: as we always use borderWidth :1 in Input I have a different style in personalinfo page. All inputs have borderWidth: 1  and phoneNumber has another so the phone number's border differs from others

File: package.json (hunk 4 above)
Comment (arlovaalina) [reply package.json:49]: OK, not it's clear :)

File: src/components/PhoneNumberInput/PhoneNumberInput.styles.js (hunk 2 above)
Comment (arlovaalina) [reply src/components/PhoneNumberInput/PhoneNumberInput.styles.js:36]: In fact, it is not good to use 1 as width. It depends on physical screen resolution. In some cases width of 1 can be smaller than 1 physical pixel and there will be visual defects. It is better to use hairlineWidth everywhere.

**Review by arlovaalina — APPROVED** [review]

**Pull request merged**